source/common/umf_pools/disjoint_pool.cpp

```diff
@@ -934,8 +934,6 @@ void *DisjointPool::aligned_malloc(size_t size, size_t alignment) {
 
 size_t DisjointPool::malloc_usable_size(void *) {
     // Not supported
-    assert(false);
```

Comment:
why you just deleted this line instead of replace with `umf::getPoolLastStatusRef<DisjointPool>() = UMF_RESULT_ERROR_NOT_SUPPORTED;`?

Reply:
I didn't want to use it here, since this function is not an allocation function. And if it returns zero, we can tell that it's not supported.

Reply:
`malloc_usable_size` returns 0 for null pointers and zero-length allocations are technically a thing. So you might not necessarily be able to disambiguate between a valid call and an error - but do we need to? I'd argue that we don't.